amf-api-contract/shared/src/main/scala/amf/apicontract/internal/spec/async/parser/context/syntax/Async21Syntax.scala

```diff
@@ -1,8 +1,14 @@
 package amf.apicontract.internal.spec.async.parser.context.syntax
 
-import amf.apicontract.internal.spec.async.parser.bindings.Bindings.Mercure
+import amf.apicontract.internal.spec.async.parser.bindings.Bindings.{IBMMQ, Mercure}
 import amf.shapes.internal.spec.common.parser.SpecSyntax
 
 object Async21Syntax extends SpecSyntax {
-  override val nodes: Map[String, Set[String]] = add(Async20Syntax.nodes, "bindings", Set(Mercure))
+  override val nodes: Map[String, Set[String]] =
+    add(
+      Async20Syntax.nodes,
+      "bindings" -> Set(Mercure),
+      Mercure    -> Set(),
+      "ibmmq"    -> Set()
```

Comment:
if this is a mercure bindings commit, add only mercure binding related things.

Reply:
done, reorganized commits